collections/_catalog/deployment/book-info.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: item
+name: Book Info
+type: Deployment
+compatibility:
+    - Istio
+patternId: MESHERY033
+image: /images/patterns/service-mesh.svg
+patternInfo: "This design deploys Open Service Mesh's 'Book Store' sample application into the `bookstore` namespace. Sample applications from one service mesh project can be deployed to different service mesh. A running Istio control plane in a Kubernetes cluster. "
+patternCaveats: ""
+URL: "https://raw.githubusercontent.com/service-mesh-patterns/service-mesh-patterns/master/samples/bookInfoPatternForCatalog.yaml"
```

Comment:
As @acald-creator mentioned, this item seems to be the BookInfo sample app by istio and not Bookstore by OSM (By the looks, they look different). Let me know if I'm missing something

Reply:
@gr455 @acald-creator I copied the one that @Revolyssup added as catalog content, let me know if I am missing something 😅

Reply:
@warunicorn19 It just needs to be updated to reflect that it is a BookInfo sample by `Istio` not Microsoft `OSM`.

This needs to be changed on the original file as well.

Reply:
@acald-creator will you take this PR/issue forward? @warunicorn19 has stepped away from contributing code.